frontstage/views/surveys/help/surveys_help.py

```diff
@@ -187,13 +234,15 @@ def _send_new_message(subject, party_id, survey, business_id):
     return response
 
 
-def get_subject_and_breadcrumbs_title(option, uri):
-    """Gets the subject line for the secure message, the title of the breadcrumbs for sub options """
-    if option == 'do-not-have-specific-figures':
-        return 'I don’t have specific figures for a response', \
-               "Help completing this survey", \
-               "I don’t have specific figures for a response"
-    if option == 'unable-to-return-by-deadline':
-        return 'I’m unable to return the data by the deadline', \
-               "Help completing this survey", \
-               "I’m unable to return the data by the deadline"
+def _inside_legal_basis(legal_basis):
+    """
+    Returns true if the legal basis matches as mandatory.
+
+            Parameters:
```

Comment:
This isn't the restructured text format for the docstrings (I can't remember the name for this style, might be the google one?).  Can you change it so it's like the others in this repo?

Reply:
Doesn't have a reference point in this repo, but I think what you meant. Updated

Reply:
There is no reference in this project, but I think I understand what you mean.